controllers/episodes.ts

```diff
@@ -52,11 +58,13 @@ export const get_episodes_by_podcast_id = asyncHandler(
     const episodes = await prisma.episodeInfo.findMany({
       where: {
         podcastId,
-        ...(unpublished ? {} : { publishedAt: { lte: new Date() } }),
+        ...(unpublished
+          ? { isDraft: false }
+          : { publishedAt: { lte: new Date() } }),
```

Comment:
Does this also need the isDraft conditional?

```suggestion
          : { isDraft: false, publishedAt: { lte: new Date() } }),
```

Reply:
yup i missed this one, good catch